### PR TITLE
Add RDFa tags for Schema.org

### PR DESCRIPTION
**Pull request by prtksxna**
Bug: [T177812](https://phabricator.wikimedia.org/T177812)

### CODE REVIEW

**Comment by prtksxna** [comment]
Adding the recommendation of @j4n-co from https://phabricator.wikimedia.org/T177812#3984992. 

Could you confirm that this is ok to add on all the pages?

File: index.html [hunk 1]
```diff
@@ -110,15 +110,15 @@ <h2>How to participate</h2>
 			</div>
 		</div>
 	</div>
-	<footer id="footer" class="footer">
+	<footer id="footer" class="footer" itemscope="" itemtype="http://schema.org/DigitalDocument">
```
Comment (Volker-E) [inline index.html]: 1. That should be further up in the DOM. 
2. You're using Microdata now, I thought we're going for RDFa Lite?

**Comment by prtksxna** [comment]
Sorry about missing your previous comment!

> It could be also considered to add two typeof attributes with values WebSite and WebPage to head and body respectively.

Interesting idea. Could you point me to documentation that talks about multiple `typeof` attributes on a single document and its affects on consumers of the data (if any).

**Comment by Volker-E** [comment]
@prtksxna [This is one example](view-source:http://boffinhouse.com/) in the wild, although, there are no further data items underneath `itemscope itemtype="http://schema.org/WebSite"`.
`https://schema.org/WPFooter` is a separate schema item type, so it's not clear how much sense the addition of `WebSite` makes at all.

Here's another example showing `WebPage`, `Article` & `WPSideBar` side-by-side:
https://stackoverflow.com/questions/22824275/schema-org-property-to-link-items-to-webpage

**Comment by Volker-E** [comment]
@prtksxna [This is one example](view-source:http://boffinhouse.com/) in the wild, although, there are no further data items underneath `itemscope itemtype="http://schema.org/WebSite"`.
`https://schema.org/WPFooter` is a separate schema item type, so it's not clear how much sense the addition of `WebSite` makes at all.

Here's another example showing `WebPage`, `Article` & `WPSideBar` side-by-side:
https://stackoverflow.com/questions/22824275/schema-org-property-to-link-items-to-webpage

**Comment by prtksxna** [comment]
Thanks for the links @Volker-E. I am thinking we should keep it simple for now since there is little or unknown benefit to the added markup (even for what is being currently added). Is it ok if we add more as and when more information surfaces?

**Comment by Volker-E** [comment]
@prtksxna Yeah, seems fine to me. Nevertheless putting `WebSite` on `body` feels semantically wrong. I'd rather go with `WebPage` then. 
I'd recommend, as I stated on task, to throw final source code at https://search.google.com/structured-data/testing-tool

They seem to have a problem with `rel=noopener` which I've reported.
It also became clear that
- `author` needs a `typeof="Organization"` and
- Phab link could make use of `property="discussionUrl"`

**Comment by prtksxna** [comment]
I just noticed that in #102 we are alternating the position of the Privacy Policy link (once after the WMF link, once after). Should fix along with this patch. I'd prefer having it _after_ the WMF link.

**Comment by Volker-E** [comment]
Feels visually weird to me to have Wikimedia Project uppercase above Privacy policy, as in changing font weight. The heaviest should be the final one in my opinion.

**Comment by prtksxna** [comment]
@Volker-E, not sure what you're proposing, I don't see any change in weight. 

I am guessing you'd prefer the WMF link to be at the bottom?

**Pull request merged**